src/daft-core/src/datatypes/logical.rs

```diff
@@ -62,6 +62,9 @@ impl<L: DaftLogicalType, P: DaftArrayType> LogicalArrayImpl<L, P> {
 
 macro_rules! impl_logical_type {
     ($physical_array_type:ident) => {
+        // Clippy triggers false positives here for the MapArray implementation
+        // This is added to suppress the warning
+        #[allow(clippy::len_without_is_empty)]
```

Comment:
Noted -- I guess we never found the issue?

Reply:
nope :(